sky/clouds/kubernetes.py

```diff
@@ -368,10 +438,10 @@ def accelerator_in_region_or_zone(self,
                                       acc_count: int,
                                       region: Optional[str] = None,
                                       zone: Optional[str] = None) -> bool:
-        # TODO(romilb): All accelerators are marked as not available for now.
+        # TODO(romilb): All accelerators are marked as available for now.
         #  In the future, we should return false for accelerators that we know
         #  are not supported by the cluster.
```

Comment:
Should we do the similar check as the code starting at L331? We can probably refactor the checking out and call it here as well?